tests/kernel/threads/thread_apis/src/main.c

```diff
@@ -167,7 +168,7 @@ void test_thread_name_user_get_set(void)
 	char too_small[2];
 
 	/* Some memory-related error cases for k_thread_name_set() */
-	ret = k_thread_name_set(NULL, (const char *)0xFFFFFFF0);
+	ret = k_thread_name_set(NULL, (const char *)_image_rom_start);
```

Comment:
I assumed that the _image_rom_start is equally "accepted nonsense" as 0xFFFFFFF0...

Reply:
Update: the test case is not guaranteed to run for Cortex-M in Non-Secure domain; the point of this test case is to cause a bus fault in the system (reading from non-existing address) but in NS domain, this will trigger Secure Fault and this is handled outside the zephyr kernel.